content/doc/book/system-administration/administering-jenkins.adoc

```diff
@@ -0,0 +1,101 @@
+---
+layout: section
+title: Administering Jenkins
+---
+
+== `JENKINS_HOME` directory
+Jenkins needs some disk space to perform builds and keep archives.
+You can check this location from the configuration screen of Jenkins.
```

Comment:
Is this a bit vague?  Maybe something like "The current location is displayed in the ?? box on the Jenkins configuration screen"